src/main/scala/day5/Calculator.scala

```diff
@@ -0,0 +1,39 @@
+package day5
+
+class Calculator {
+  def sum(inputIntOne: Int, inputIntTwo: Int): Any = {
+    val inputIntOneLong = inputIntOne.toLong
+    val inputIntTwoLong = inputIntTwo.toLong
+    inputIntOneLong + inputIntTwoLong match {
+      case resultMaxValueChecker: Long if resultMaxValueChecker > Integer.MAX_VALUE => "result out of bounds"
+      case resultMinValueChecker: Long if (-resultMinValueChecker) > Integer.MAX_VALUE => "result out of bounds"
```

Comment:
these two cases are duplicated below `case resultMaxValueChecker: Long if resultMaxValueChecker > Integer.MAX_VALUE => "result out of bounds"
      case resultMinValueChecker: Long if (-resultMinValueChecker) > Integer.MAX_VALUE => "result out of bounds"` 

you could replace them with a case that would call a method with the value you want to check and then get the string back, you're duplicating quite a bit of functionality here